addons/website_mass_mailing/views/snippets_templates.xml

```diff
@@ -81,79 +81,35 @@
 </template>
 
 <!-- Extend default mass_mailing snippets with website feature -->
-<template id="social_links">
+<template id="mass_mailing.social_links">
     <t t-if="not website">
         <t t-set="website" t-value="request.env['website'].search([], limit=1)"/>
```

Comment:
@Gorash Why did you add this here ? Any reason ?

Reply:
Remember for mass mailing, but it may be a remnant of refactoring and revert.